exec/plugin-commands-script-runners/src/dlx.ts

```diff
@@ -159,23 +160,30 @@ function scopeless (pkgName: string): string {
 function findCache (pkgs: string[], opts: {
   cacheDir: string
   dlxCacheMaxAge: number
+  registries: Record<string, string>
 }): { cacheLink: string, prepareDir: string | null } {
-  const dlxCommandCacheDir = createDlxCommandCacheDir(opts.cacheDir, pkgs)
+  const dlxCommandCacheDir = createDlxCommandCacheDir(opts.cacheDir, pkgs, opts.registries)
```

Comment:
I'd use the same argument positions as in findCache
```suggestion
  const dlxCommandCacheDir = createDlxCommandCacheDir(pkgs, opts)
```